tools/autodeployment/slack_app/index.js

```diff
@@ -0,0 +1,54 @@
+require('dotenv').config();
+const axios = require('axios').default;
+
+const { SLACK_MESSAGE_HOOK } = process.env;
+
+const prodStatus = () => {
+  let alarm = false;
+
+  setInterval(async () => {
+    let prodResponse = {};
+
+    try {
+      prodResponse = await axios.get('https://telescope.cdot.systems/posts', {
+        method: 'HEAD',
+        timeout: 3000,
+      });
+    } catch (error) {
+      if (!alarm) {
+        const downDate = new Date().toTimeString();
+        axios.post(SLACK_MESSAGE_HOOK, {
+          response_type: 'in_channel',
+          text: `\n:rotating_light: PRODUCTION IS DOWN. :rotating_light:\n`,
```

Comment:
Here, instead of hardcoding `Prod` or `Dev`, you can use [DEPLOY_TYPE](https://github.com/Seneca-CDOT/telescope/blob/38be67c1c0692df619e21174f3657eb661a47464/tools/autodeployment/env.example#L12)'s value to know which server is running this code.